package.json

```diff
@@ -164,6 +164,12 @@
       "import": "{ app, authentication, pages }",
       "limit": "57 KB"
     },
+    {
+      "brotli": false,
```

Comment:
Does this run automatically now? Or do we need to add another step to the pipeline for it?

Reply:
It runs automatically whenever teamsJS builds. It's part of the build command.